sdk/go/common/apitype/service_test.go

```diff
@@ -27,7 +27,9 @@ func TestCapabilities(t *testing.T) {
 		t.Parallel()
 		actual, err := CapabilitiesResponse{}.Parse()
 		assert.NoError(t, err)
-		assert.Equal(t, Capabilities{}, actual)
+		assert.Equal(t, Capabilities{
```

Comment:
Here again you're adding code to the existing functionality. From now on, every new test will have to add the same thing?

Reply:
Yes, its useful to assert the state of this map in the tests if we go down this road.